src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -248,22 +248,41 @@ public virtual async Task<T> GetConfigurationAsync(CancellationToken cancel)
                 {
                     if (_refreshRequested)
                     {
-                        // Log as manual because RequestRefresh was called
-                        TelemetryClient.IncrementConfigurationRefreshRequestCounter(
-                            MetadataAddress,
-                            TelemetryConstants.Protocols.Manual);
+                        _refreshRequested = false;
+
+                        try
+                        {
+                            // Log as manual because RequestRefresh was called
+                            TelemetryClient.IncrementConfigurationRefreshRequestCounter(
+                                MetadataAddress,
+                                TelemetryConstants.Protocols.Manual);
+                        }
+#pragma warning disable CA1031 // Do not catch general exception types
+                        catch
+                        { }
+#pragma warning restore CA1031 // Do not catch general exception types
 
                         UpdateCurrentConfiguration();
-                        _refreshRequested = false;
                     }
-                    else
+                    else if (SyncAfter <= _timeProvider.GetUtcNow())
                     {
-                        TelemetryClient.IncrementConfigurationRefreshRequestCounter(
+                        try
+                        {
+                            TelemetryClient.IncrementConfigurationRefreshRequestCounter(
                             MetadataAddress,
                             TelemetryConstants.Protocols.Automatic);
```

Comment:
```suggestion
                                MetadataAddress,
                                TelemetryConstants.Protocols.Automatic);
```
For consistent indentation

Reply:
@westin-m thanks, fixed.